airflow/providers/neo4j/example_dags/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Do we need this file? `example_dags` is never "imported" so we don't need to make this a module.

Is this what we do for other providers?